code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -130,14 +130,15 @@
 proc/flame_radius(radius = 1, turf/T, burn_intensity = 25, burn_duration = 25, burn_damage = 25, fire_stacks = 15, int_var = 0.5, dur_var = 0.5, colour = "red") //~Art updated fire.
 	if(!T || !isturf(T))
 		return
-	radius = CLAMP(radius, 1, 7) //Sterilize inputs
+	radius = CLAMP(radius, 1, 50) //Sanitize inputs
```

Comment:
50 is too big.

Reply:
That's just sanitization in case something were to call that proc with shitty radius